distribution/client/public/index.html

```diff
@@ -30,6 +30,9 @@
             .connected {
                 background-color: #006a27;
             }
+            pre {
+                white-space: pre-wrap;
+            }
```

Comment:
I'm inclined to think that this might not be the right fix: the `index.js` inserts elements using `document.createElement('pre')`, when these are actually messages we probably want wrapping and default formatting to act on, instead of preformatted code (like program source code).
I.e. inserting there `<p>` for instance, and use css to format like we would like to appear.